### PR TITLE
Add SPDX header - batch #2

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* Resolves: follow-up for #45244

## Summary

Add further SPDX header or reformat existing headers for the converter to work

---
## Before
* files with copyright information: 8292 / 10921
* files with license information: 4242 / 10921

## After
* files with copyright information: 8687 / 10921
* files with license information: 5105 / 10921

## TODO

- [x] Migrate further js resources

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by susnux — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
@susnux, can you please re-review. I triggered the compile and before that pushed all headers for `core/js` 🎉

**Comment by AndyScherzinger** [comment]
Any hints as to why the changed/now SPDX header won't get copied over? see i.e. https://github.com/nextcloud/server/pull/45252/commits/95f1c0918e682713b5671e778131f846302fd135#diff-21c6a91f3d35ef70889606c918fa3ee8d5dbd41e0983ccba0514c49263065ba5

File: core/css/inputs.scss [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
- *
- * @license GNU AGPL version 3 or any later version
+ * @copyright 2016 John Molakvoæ <skjnldsv@protonmail.com>
+ * @copyright 2016 Morris Jobke <hey@morrisjobke.de>
+ * @copyright 2016 Joas Schilling <coding@schilljs.com>
+ * @copyright 2016 Julius Haertl <jus@bitgrid.net>
+ * @copyright 2016 jowi <sjw@gmx.ch>
+ * @copyright 2015 Joas Schilling <nickvergessen@owncloud.com>
+ * @copyright 2015 Hendrik Leppelsack <hendrik@leppelsack.de>
+ * @copyright 2014-2017 Jan-Christoph Borchardt <hey@jancborchardt.net>
```
Comment (susnux) [inline core/css/inputs.scss:9]: Should this not be `SPDX-FileCopyrightText` instead?

Comment (AndyScherzinger) [reply core/css/inputs.scss:9]: No, this is just a change so the actual converter script will pick up the comments correctly.

But yeah. I can also just run the script and commit the SPDX headers 😂

Comment (AndyScherzinger) [reply core/css/inputs.scss]: Fixed via https://github.com/nextcloud/server/pull/45252/commits/c9e56795f9f63873575a3e786b9f402331629c86

**Comment by susnux** [comment]
@AndyScherzinger despite of https://github.com/nextcloud/server/pull/45259 I pushed a commit that fixes extracting the comments (added `@copyright` and `@spdx` as pattern for comments to extract).

**Comment by AndyScherzinger** [comment]
@susnux sounds good, however the SPDX headers don't start with an `@`, not sure of that is relevant. However some license files seem to be gone completely now, not sure if that is also related to the change you did. best case we "just" have `.license` files for all files in the dist folder

File: .github/pull_request_template.md.license
Comment (nickvergessen) [inline .github/pull_request_template.md.license:1]: Fail content

Comment (AndyScherzinger) [reply .github/pull_request_template.md.license:1]: nice catch 👍 

fixed via https://github.com/nextcloud/server/pull/45252/commits/1b06a005b3dd73504eff8080cb88a9a5f3743083

**Review by ChristophWurst — APPROVED** [review]
Squash?

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
image

**Comment by AndyScherzinger** [comment]
Commits so they still "make sense" and are specific to their actual character of change

**Review by artonge — APPROVED** [review]

**Pull request merged**